router.go

```diff
@@ -5,6 +5,18 @@ import (
 	"net/http"
 )
 
+const (
+	GET     = "GET"
+	HEAD    = "HEAD"
+	POST    = "POST"
+	PUT     = "PUT"
+	PATCH   = "PATCH"
+	DELETE  = "DELETE"
+	CONNECT = "CONNECT"
+	OPTIONS = "OPTIONS"
+	TRACE   = "TRACE"
+)
```

Comment:
This constants are actually present on the net/http package, I would recommend to use them instead